### PR TITLE
W-16865440 Exchange -  AsyncAPI 2.6  with Avro schema release jh

### PR DESCRIPTION
**Pull request by JennyHajee**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 1]
```diff
@@ -197,8 +196,23 @@ To create a ruleset asset:
...
-. If you want to edit the advanced settings, *GroupId*, *AssetId*, and *Version* (GAV), click *Advanced*. 
+. To upload, click *Choose file*.
+. To edit the advanced settings, *GroupId*, *AssetId*, and *Version* (GAV), click *Advanced*. 
++
+Exchange generates the group ID, asset ID, and version for you, and you can change these values as needed. Use the advanced settings to change the asset version.
+. Select an option for *Lifecycle state*. The default is *Stable*.
+. Select *Publish*.
+
+== Create an AsyncAPI Asset
+An AsyncAPI is an AsyncAPI specification file that specifies an event-driven API. You can create AsyncAPIs with Avro schemas. The file types for AsyncAPIs with Avro schemas are JSON (.json) and AVSC (.avsc).
```
Comment (lbauret) [inline modules/ROOT/pages/to-create-an-asset.adoc]: File types for AsyncAPIs are only `.json` and `.yaml`. `.avsc` is only for AVRO fragments

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 2]
```diff
@@ -231,12 +245,14 @@ If you have contributor or admin access to an asset, you can add a new version f
 . In Exchange, open the asset list and select the asset.
 . Click *Add version*. 
 . For API assets, follow the steps for the corresponding API asset type:
-* REST API - RAML: Provide a RAML API specification file. * RAML specifications must be a RAML file (.raml).
-* REST API - OAS: Provide an OAS API specification file. *OAS specifications can be either a YAML (.yaml) or JSON (.json) file. Exchange supports OAS 2.0 and OAS 3.0 specifications.
+* REST API - RAML: Provide a RAML API specification file.  RAML specifications must be a RAML file (.raml).
+* REST API - OAS: Provide an OAS API specification file.  OAS specifications can be either a YAML (.yaml) or JSON (.json) file. Exchange supports OAS 2.0 and OAS 3.0 specifications.
 * SOAP API - WSDL: Provide a WSDL API specification file. SOAP specifications file can be either a WSDL (.wsdl) or XML (.xml) file.
 * API Spec Fragment - RAML : Provide an API Fragment RAML specification file. Fragment specifications must be a RAML file (.raml).
+* AsyncAPI specifications with Avro schemas can be a JSON (.json) or AVSC (.avsc) file.
```
Comment (lbauret) [inline modules/ROOT/pages/to-create-an-asset.adoc]: Same as above

**Review by riajoshi24 — DISMISSED** [review]

**Review by lbauret — DISMISSED** [review]
LGTM!

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 3]
```diff
@@ -231,12 +246,15 @@ If you have contributor or admin access to an asset, you can add a new version f
 . In Exchange, open the asset list and select the asset.
 . Click *Add version*. 
 . For API assets, follow the steps for the corresponding API asset type:
-* REST API - RAML: Provide a RAML API specification file. * RAML specifications must be a RAML file (.raml).
-* REST API - OAS: Provide an OAS API specification file. *OAS specifications can be either a YAML (.yaml) or JSON (.json) file. Exchange supports OAS 2.0 and OAS 3.0 specifications.
+* REST API - RAML: Provide a RAML API specification file.  RAML specifications must be a RAML file (.raml).
+* REST API - OAS: Provide an OAS API specification file.  OAS specifications can be either a YAML (.yaml) or JSON (.json) file. Exchange supports OAS 2.0 and OAS 3.0 specifications.
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/to-create-an-asset.adoc]:
```suggestion
* REST API - OAS: Provide an OAS API specification file. OAS specifications can be either a YAML (.yaml) or JSON (.json) file. Exchange supports OAS 2.0 and OAS 3.0 specifications.
```

**Review by bjoson-mulesoft — DISMISSED** [review]

**Review by bjoson-mulesoft — DISMISSED** [review]

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 4]
```diff
@@ -231,12 +246,15 @@ If you have contributor or admin access to an asset, you can add a new version f
 . In Exchange, open the asset list and select the asset.
 . Click *Add version*. 
 . For API assets, follow the steps for the corresponding API asset type:
-* REST API - RAML: Provide a RAML API specification file. * RAML specifications must be a RAML file (.raml).
-* REST API - OAS: Provide an OAS API specification file. *OAS specifications can be either a YAML (.yaml) or JSON (.json) file. Exchange supports OAS 2.0 and OAS 3.0 specifications.
+* REST API - RAML: Provide a RAML API specification file.  RAML specifications must be a RAML file (.raml).
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/to-create-an-asset.adoc]:
```suggestion
* REST API - RAML: Provide a RAML API specification file. RAML specifications must be a RAML file (.raml).
```

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**